src/Symfony/Component/Form/Extension/Core/Type/CollectionType.php

```diff
@@ -120,6 +120,7 @@ public function configureOptions(OptionsResolver $resolver)
             'prototype_name' => '__name__',
             'entry_type' => TextType::class,
             'entry_options' => [],
+            'prototype_options' => [],
```

Comment:
please add also `setAllowedTypes('prototype_options', 'array')` definition

Reply:
Thanks, I added it.